components/intl.rst

```diff
@@ -376,6 +377,39 @@ You can also check if a given currency code is valid::
 
     The ``Currencies`` class was introduced in Symfony 4.3.
 
+Timezones
+~~~~~~~~~
+
+The ``Timezones`` class provides access to the name and values of all timezones::
+
+    use Symfony\Component\Intl\Timezones;
+
+    \Locale::setDefault('en');
+
+    $timezones = Timezones::getNames();
+    // ('timezoneName' => 'timezoneValue')
+    // => ['America/Eirunepe' => 'Acre Time (Eirunepe)', 'America/Rio_Branco' => 'Acre Time (Rio Branco)', ...]
+
+    $timezone = Timezones::getName('Africa/Nairobi');
+    // => 'East Africa Time (Nairobi)'
+
+All methods accept the translation locale as the last, optional parameter,
+which defaults to the current default locale::
+
+    $timezones = Timezones::getNames('de');
+    // => ['America/Eirunepe' => 'Acre-Zeit (Eirunepe)', 'America/Rio_Branco' => 'Acre-Zeit (Rio Branco)', ...]
+
+    $timezone = Timezones::getName('Africa/Nairobi', 'de');
+    // => 'Ostafrikanische Zeit (Nairobi)'
+
+You can also check if a given timezone name is valid::
```

Comment:
`timezone ID` or `timezone identifier`

Reply:
Thanks for reviewing! I made this change in `master` directly: ca9db00643410845e8078f879251eb3789825263